crates/shared/src/bad_token/token_owner_finder.rs

```diff
@@ -8,25 +8,21 @@ use {
     self::{
         blockscout::BlockscoutTokenOwnerFinder,
         liquidity::{
-            BalancerVaultFinder,
-            FeeValues,
-            UniswapLikePairProviderFinder,
-            UniswapV3Finder,
+            BalancerVaultFinder, FeeValues, UniswapLikePairProviderFinder, UniswapV3Finder,
         },
     },
     crate::{
         arguments::{display_list, display_option, display_secret_option},
         bad_token::token_owner_finder::{
             ethplorer::EthplorerTokenOwnerFinder,
             solvers::{
-                solver_api::SolverConfiguration,
-                solver_finder::AutoUpdatingSolverTokenOwnerFinder,
+                solver_api::SolverConfiguration, solver_finder::AutoUpdatingSolverTokenOwnerFinder,
             },
             token_owner_list::TokenOwnerList,
         },
         baseline_solver::BaseTokens,
         ethcontract_error::EthcontractErrorType,
-        ethrpc::{Web3, Web3CallBatch, MAX_BATCH_SIZE},
```

Comment:
Could mark Web3CallBatch as deprecated to get a clippy warning when it's used outside of the `ethrpc` crate.

Reply:
I think there are a few other places where we also manually batch. I can add the clippy warning in a follow up PR where I get rid of those occurrences.